epaint/src/color.rs

```diff
@@ -828,7 +828,7 @@ impl From<Hsva> for HsvaGamma {
 // ----------------------------------------------------------------------------
 
 /// Cheap and ugly.
-/// Made for graying out disabled `Ui`:s.
+/// Made for graying out disabled [`Ui`]:s.
```

Comment:
How should I qualify this one? Since the `egui` crate itself is not imported here I seem to be unable to please cargo without importing it.

Reply:
I don't think `epaint` can point to `egui`, since it is not a dependency except by linking explicitly to https://docs.rs/egui/latest/egui/struct.Ui.html